source/gameengine/Ketsji/KX_VehicleWrapper.cpp

```diff
@@ -407,4 +409,11 @@ int KX_VehicleWrapper::pyattr_set_ray_mask(void *self, const struct KX_PYATTRIBU
 	return PY_SET_ATTR_SUCCESS;
 }
 
+PyObject *KX_VehicleWrapper::pyattr_get_speed(void *self, const struct KX_PYATTRIBUTE_DEF *attrdef)
+{
+	KX_VehicleWrapper *wrapper = static_cast<KX_VehicleWrapper*>(self);
+	MT_Scalar speed = std::abs(wrapper->m_vehicle->GetCurrentSpeed()); // No negative speeds
```

Comment:
Yes why not? It's speed. It would be very weird if a speedometer showed -50 Km/h :stuck_out_tongue: